models/comments.model.js

```diff
@@ -47,3 +47,32 @@ exports.insertComment = async (body, article_id) => {
   ]);
   return rows[0];
 };
+
+exports.fetchCommentByCommentId = async (comment_id) => {
+  try {
+    let commentQuery = `SELECT *
+      FROM
+          comments
+      WHERE
+          comments.comment_id = $1;`;
+    const { rows } = await db.query(commentQuery, [comment_id]);
+
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "comment does not exist" });
+    }
+    return rows;
+  } catch (error) {
+    throw error;
+  }
+};
+
+exports.removeCommentById = async (comment_id) => {
+  try {
+    let commentQuery = `DELETE FROM comments
+      WHERE
+          comments.comment_id = $1;`;
+    await db.query(commentQuery, [comment_id]);
+  } catch (error) {
```

Comment:
Having a catch here and in the model is not necessary as if theres an error in the model, it will be caught in the controller anyway